processor/resourcedetectionprocessor/internal/aws/eks/metadata.yaml

```diff
@@ -11,6 +11,10 @@ resource_attributes:
     description: The cloud.platform
     type: string
     enabled: true
+  cloud.account.id:
+    description: The cloud account id
+    type: string
+    enabled: true
```

Comment:
New attributes should be added as disabled by default. Otherwise, it's considered a breaking change